mission_yaml/SEA67_M46.yml

```diff
@@ -0,0 +1,626 @@
+  institution: Voice of the Ocean Foundation
+  license: Creative Commons Attribution 4.0 (https://creativecommons.org/licenses/by/4.0/)
+    This study used data collected and made freely available by Voice of the Ocean Foundation
+    (https://voiceoftheocean.org) accessed from https://erddap.observations.voiceoftheocean.org/erddap/index.html
+  format_version: IOOS_Glider_NetCDF_v2.0.nc
+  glider_model: SeaExplorer
+  glider_instrument_name: seaexplorer
+  keywords: "AUVS, Autonomous Underwater Vehicles, Oceans, Ocean Pressure,
+             Water Pressure, Ocean Temperature, Water Temperature,
+             Salinity/Density, Conductivity, Density, Salinity"
+  keywords_vocabulary: GCMD Science Keywords
+  metadata_link: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
+  Metadata_Conventions: CF-1.6, Unidata Dataset Discovery v1.0
+  naming_authority: Voice of the Ocean Foundation
+  platform:    "SeaExplorer Glider"
+  processing_level: "Data provided as is with no expressed or implied
+                     assurance of quality assurance or quality control. L0"
+  publisher_email: callum.rollo@voiceoftheocean.org
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  sea_name: Baltic
+  transmission_system: IRIDIUM
+  glider_name: Marviol
+  glider_serial: "67"
+  wmo_id: "6801507"
+  comment: 'deployment and recovery in Bornholm'
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin, Gunnar Johnsson, Andrew Birkett, Chiara Monforte
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician, Glider Technician, Glider Technician, Glider Pilot
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://observations.voiceoftheocean.org
+  deployment_id: "46"
+  deployment_name: SAMBA
+  project: SAMBA
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of SAMBA continuous monitoring'
+
+
+glider_devices:
+  ctd:
+    make: RBR
+    model: legato
+    serial: '210555'
+    long_name: RBR legato CTD
+    make_model:  RBR legato CTD
+    factory_calibrated: "Yes"
+    calibration_date: "2022-05-19"
+  optics:
+    make: Wetlabs
+    model: FLBBPC
+    serial: '7313'
+    long_name: Wetlabs ECO Puck FLBBPC
+    make_model: Wetlabs FLBBPC
+    factory_calibrated: "Yes"
+    calibration_date: "2022-01-11"
+    calibration_parameters: {Chl_DarkCounts: 44, Chl_SF: 0.0072, PC_DarkCounts: 51, PC_SF: 0.0423, SIG_DarkCounts: 48, SIG_SF: 1.899E-06}
+  oxygen:
+    make: JFE Advantech
+    model: AROD-FT
+    serial: '0039'
```

Comment:
logbook says we have rinko 0069 on this glider, can you check @MOchiara ?

Reply:
Yeah I know, in configs on the server it says we have 0039 which has just been 2-point calibrated